gym/lib/gym/options.rb

```diff
@@ -228,6 +228,12 @@ def self.plain_options
                                      verify_block: proc do |value|
                                        UI.user_error!("File not found at path '#{File.expand_path(value)}'") unless File.exist?(value)
                                      end),
+        FastlaneCore::ConfigItem.new(key: :xcode_preference_plist_path,
```

Comment:
We don't! I didn't realize you could do all this wonderful stubbing in Ruby so we don't need to inject anything via the options.
#rubyNoob